images/macos/helpers/Xcode.Installer.psm1

```diff
@@ -175,6 +175,16 @@ function Build-XcodeSymlinks {
     }
 }
 
+function Rebuild-Xcode-LaunchServicesDb {
```

Comment:
Replace function name to powershell style Verb-Noun like `Rebuild-XcodeLaunchServicesDb`